terraform/modules/kubernetes/monitoring.tf

```diff
@@ -115,3 +115,45 @@ resource "azurerm_monitor_metric_alert" "redis_memory" {
     ]
   }
 }
+
+data "azurerm_monitor_diagnostic_categories" "postgres" {
+  count = var.deploy_azure_backing_services ? 1 : 0
```

Comment:
In the terraform modules these are controlled by `azure_enable_monitoring` should we keep it the same, so we can control when we turn it on in each environment? @RMcVelia thoughts?

Reply:
makes sense, guess that is  var.enable_alerting as per the count statements for the metrics in the monitoring.tf